### PR TITLE
Add new recipe DeBreak

### PR DESCRIPTION
**Pull request by czc**
new file:   recipes/debreak/meta.yaml

Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by czc** [comment]
@ BiocondaBot please add label

**Comment by czc** [comment]
@BiocondaBot hello

**Comment by czc** [comment]
@BiocondaBot please add label

File: recipes/debreak/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    url: https://github.com/ChongLab/DeBreak/archive/refs/tags/debreak-v{{ version }}.tar.gz
+    sha256: 6162786bc60dc995d25790da6f806ed25bcfa50a7b10dc6e8404cacba7ce36df
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python <3
```
Comment (bgruening) [inline recipes/debreak/meta.yaml:17]: Can we get a Python3 version of this tool? Python2 is EoL :(

Comment (czc) [reply recipes/debreak/meta.yaml:17]: Thanks very much for reviewing and for the suggestion. The conda environment has been tested locally. The python3 version is planned to be updated. Could we release this build and upload future versions later? Thank you.

File: recipes/debreak/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - python =2.7
+    - minimap2 =2.15
+    - pysam =0.16.0.1
+    - samtools =1.9
+    - wtdbg =2.5
```
Comment (dpryan79) [inline recipes/debreak/meta.yaml:21]: Do we really need these exact versions of these dependencies and now say "minimap2 >=2.15"?

Comment (czc) [reply recipes/debreak/meta.yaml:21]: We have not tested the newer versions. Should be Okay. But it's not guaranteed.

**Review by dpryan79 — APPROVED** [review]

**Pull request merged**